From 2782f0ab3fe5850c7e5a7bd2a67b82b3d5bf3616 Mon Sep 17 00:00:00 2001
From: Alex Zuepke <alex.zuepke@minervasys.tech>
Date: Tue, 19 Nov 2024 17:09:14 +0100
Subject: [PATCH 1/3] Jailhouse: disable cpuidle driver on Orin when VHE is
 disabled

The Nvidia cpuidle driver crashes with an exception when VHE is disabled.

Signed-off-by: Alex Zuepke <alex.zuepke@minervasys.tech>
---
 nvidia/drivers/cpuidle/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvidia/drivers/cpuidle/Makefile b/nvidia/drivers/cpuidle/Makefile
index fc12612c0..5a66b1b27 100644
--- a/nvidia/drivers/cpuidle/Makefile
+++ b/nvidia/drivers/cpuidle/Makefile
@@ -7,4 +7,4 @@ obj-$(CONFIG_CPU_IDLE_TEGRA19X)         += cpuidle-tegra19x.o
 obj-$(CONFIG_CPU_IDLE_TEGRA_AUTO)	+= cpuidle-tegra-auto.o
 endif
 
-obj-$(CONFIG_ARCH_TEGRA_23x_SOC)	+= cpuidle-debugfs-tegra23x.o
+#obj-$(CONFIG_ARCH_TEGRA_23x_SOC)	+= cpuidle-debugfs-tegra23x.o
-- 
2.34.1

